dbcon/execplan/windowfunctioncolumn.cpp

```diff
@@ -384,7 +384,8 @@ void WindowFunctionColumn::adjustResultType()
         fResultType = fFunctionParms[0]->resultType();
 
     if (boost::iequals(fFunctionName, "SUM") ||
-        boost::iequals(fFunctionName, "AVG"))
+        boost::iequals(fFunctionName, "AVG") ||
+        boost::iequals(fFunctionName, "AVG_DISTINCT"))
```

Comment:
What was the result data type for AVG_DISTINCT before this change?

Reply:
It was DECIMAL.